content/docs/next-release/operator.md

```diff
@@ -0,0 +1,668 @@
+---
+title: Operator for Kubernetes
+hasparent: true
+weight: 7
+---
+
+# Understanding Operators
+
+The Jaeger Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/).  Operators are pieces of software that ease the operational complexity of running another piece of software. More technically, _Operators_ are a method of packaging, deploying, and managing a Kubernetes application.
+
+A Kubernetes application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and `kubectl` (kubernetes) or `oc` (OKD) tooling. To be able to make the most of Kubernetes, you need a set of cohesive APIs to extend in order to service and manage your apps that run on Kubernetes. Think of Operators as the runtime that manages this type of app on Kubernetes.
+
+# Installing the Operator
+
+{{< danger >}}
+The Jaeger Operator version is related to the version of the Jaeger components (Query, Collector, Agent) up to the minor portion. The patch version portion does *not* follow the ones from the Jaeger components. For instance, the Operator version 1.8.1 uses the Jaeger Docker images tagged with version 1.8 by default.
```

Comment:
@jpkrohling Is this para still relevant? If we change so the operator version and backend components are version aligned? So the jaeger.version file should have a major.minor.patch number?

Reply:
Not as danger, but as info, or warning? Or even as a regular paragraph. I think we should keep the message that each Jaeger Operator version tracks one Jaeger version.